collections-csharp/HashtableInCSharp/HashtableInCSharp/HashtableDemo.cs

```diff
@@ -0,0 +1,88 @@
+﻿using System.Collections;
+
+namespace HashtableInCSharp
+{
+    public class HashtableDemo
```

Comment:
There is no instance data in this class, the whole thing can be made static, including each of the methods